src/main/java/com/blockout/Log.java

```diff
@@ -0,0 +1,29 @@
+package com.blockout;
+
+import com.minecolonies.lib.Constants;
```

Comment:
Yeah but we shouldn't add extra deps between them so we can compile blockout without minecolonies if we want